barretenberg/cpp/src/barretenberg/eccvm/eccvm_verifier.cpp

```diff
@@ -25,10 +25,14 @@ bool ECCVMVerifier::verify_proof(const ECCVMProof& proof)
     using ClaimBatch = ClaimBatcher::Batch;
 
     RelationParameters<FF> relation_parameters;
-    transcript = std::make_shared<Transcript>(proof.pre_ipa_proof);
+    if (!transcript) {
```

Comment:
Is this a test-only code path? If real usage requires that a transcript be passed in then ideally we could just enforce that via the constructors.